NewForce-Cohort-5/tabloidmvc-the-daily-debuggers
# Bg add categories

# Description

Created a form for adding categories. Also edited the nav to include a link to Categories.  Edit and delete do not work currently.

Fixes # (issue)

## Type of change


# Testing Instructions

1. Clone down the repository.
2. `git fetch -a`
3. `git checkout bg-AddCategories`
4. Navigate to the root directory for this repo (directory-to)/tabloidmvc-the-daily-debuggers
5. Start TabloidMVC.sln
6. Make sure your db is created with seed data. Run [01_TabloidMVC_Create_DB.sql](https://github.com/NewForce-Cohort-5/tabloidmvc-the-daily-debuggers/blob/main/SQL/01_TabloidMVC_Create_DB.sql) and [02_TabloidMVC_Seed_Data.sql](https://github.com/NewForce-Cohort-5/tabloidmvc-the-daily-debuggers/blob/main/SQL/02_TabloidMVC_Seed_Data.sql) queries in Visual Studio.
7. Start the server.
8. Log in as admin [admin@example.com](mailto:admin@example.com) (if not already logged in as admin).
9. Click Categories at the top of the page. 
10. Click New Category and you will be linked to a form for adding a new category.
11. Add Test to the name field and click create.
12. You should now see a list of categories,  There will be exactly 14 total.  Scroll to the very bottom of the list and you will see the new category you created with an id of 14 and a name of Test.
13. If you would like to further test this or test it through a SQL query, create a query for the database called TabloidMVC.  In your new query type: `Select * From Category` and run it.  This will show the full of the categories ordered by Id rather than alphabetical as seen from the web browser.  Look to make sure there's an entry of id 14 and a name of Test.
14. ???
15. PROFIT

***Edit and Delete do not work at this time***



# Checklist:
